stdlib/Dates/src/conversions.jl

```diff
@@ -38,26 +38,33 @@ Base.convert(::Type{Day},dt::Date) = Day(value(dt))            # Converts Date t
 
 ### External Conversions
 const UNIXEPOCH = value(DateTime(1970)) #Rata Die milliseconds for 1970-01-01T00:00:00
+localepoch() = value(DateTime(Libc.TmStruct(0))) #Rata Die for local time at UTC 1970-01-01T00:00:00
 
 """
-    unix2datetime(x) -> DateTime
+    unix2datetime(x::Real; localtime::Bool=false) -> DateTime
 
 Take the number of seconds since unix epoch `1970-01-01T00:00:00` and convert to the
-corresponding `DateTime`.
+corresponding `DateTime`. If `localtime` is `true`, then the output is in the local
+time zone, otherwise it is in UTC/GMT.
 """
-function unix2datetime(x)
+function unix2datetime(x::Real; localtime::Bool=false)
     # Rounding should match `now` below
-    rata = UNIXEPOCH + trunc(Int64, Int64(1000) * x)
+    rata = (localtime ? localepoch() : UNIXEPOCH) + trunc(Int64, Int64(1000) * x)
     return DateTime(UTM(rata))
 end
 
 """
-    datetime2unix(dt::DateTime) -> Float64
+    datetime2unix(dt::DateTime; localtime::Bool=false) -> Float64
 
 Take the given `DateTime` and return the number of seconds
 since the unix epoch `1970-01-01T00:00:00` as a [`Float64`](@ref).
+
+If `localtime` is `true`, then the number of seconds since the local epoch
+corresponding to unix epoch `1970-01-01T00:00:00` is returned.
```

Comment:
```suggestion
since the unix epoch `1970-01-01T00:00:00` (UTC) as a [`Float64`](@ref).

If `localtime` is `true`, then `dt` is interpreted as being in the local time zone,
and is otherwise interpreted as being UTC/GMT.
```